src/api/app/models/review.rb

```diff
@@ -203,11 +203,16 @@ def map_objects_to_ids(objs)
     objs.map { |obj| { "#{obj.class.to_s.downcase}_id" => obj.id } }.uniq
   end
 
-  def create_notification(params = {})
+  def notification_parameters(params = {})
     params = params.merge(_get_attributes)
     params[:id] = id
     params[:comment] = reason
     params[:reviewers] = map_objects_to_ids(users_and_groups_for_review)
+    params
+  end
+
+  def create_notification(params = {})
```

Comment:
This is private in `Comment`, why not here?

Reply:
In more recent commits it was renamed to `create_event`. Now I have also moved it to private.

Reply:
BTW, `create_event` should be public in Review model, since it is called from the Request model. Moved back to public...